### DISCUSSION TITLE
Filter PRs by conflicts/mergeability?

### DISCUSSION THREAD

**Discussion by Harvie** [community/community#36624] [Pull Requests]
Hello,
i would like to filter/seach pull requests in my repository based on the fact whether they can be merged or not.
eg.: something like this:

![image](https://user-images.githubusercontent.com/276504/196618223-3afa1999-5a8c-4027-9512-1ea1a8d254fc.png)

Can you please add such filter?

**Comment by ossanna16** [comment]
Hey there @Harvie and thank you for the feedback! I'm glad you found our community 🙂
